<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerContentsList</class>
 <widget class="QWidget" name="ViewerContentsList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>639</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>767</width>
    <height>639</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true" />
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="ViewerContentsItem" name="BookIndexItem1" native="true" />
   </item>
   <item>
    <widget class="ViewerContentsItem" name="BookIndexItem2" native="true" />
   </item>
   <item>
    <widget class="ViewerContentsItem" name="BookIndexItem3" native="true" />
   </item>
   <item>
    <widget class="ViewerContentsItem" name="BookIndexItem4" native="true" />
   </item>
   <item>
    <widget class="ViewerContentsItem" name="BookIndexItem5" native="true" />
   </item>
   <item>
    <widget class="ViewerContentsItem" name="BookIndexItem6" native="true" />
   </item>
   <item>
    <widget class="ViewerContentsItem" name="BookIndexItem7" native="true" />
   </item>
   <item>
    <widget class="ViewerContentsItem" name="BookIndexItem8" native="true" />
   </item>
   <item>
    <widget class="ViewerContentsItem" name="BookIndexItem9" native="true" />
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>28</width>
       <height>57</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewerContentsItem</class>
   <extends>QWidget</extends>
   <header>ViewerContentsItem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources />
 <connections />
</ui>